src/components/collection/Selector/Table/useCollectionsSelectorColumns.tsx

```diff
@@ -29,8 +29,10 @@ const optionalColumns = {
     } as ColumnProps,
 };
 
-const useCollectionsSelectorColumns = (
-    include?: keyof typeof optionalColumns
+export type optionalColumn = keyof typeof optionalColumns;
```

Comment:
Could the _o_ in `optionalColumn` be capitalized to align it with standard type format?

Reply:
Oh yeah... this was a `const` as first I'll update